### PR TITLE
Update spark README with container location and DNS req.

### PR DESCRIPTION
**Pull request by jayunit100**

### CODE REVIEW

**Comment by jayunit100** [comment]
Simple doc update, Goes along with #17463 and #21626

**Comment by bgrant0607** [comment]
cc @zmerlynn

**Comment by bgrant0607** [comment]
LGTM

File: examples/spark/README.md [hunk 1]
```diff
@@ -37,23 +37,25 @@ Following this example, you will create a functional [Apache
...
-This example assumes you have a Kubernetes cluster installed and
-running, and that you have installed the ```kubectl``` command line
-tool somewhere in your path. Please see the [getting
-started](../../docs/getting-started-guides/) for installation
-instructions for your platform.
+This example assumes
+
+- You have a Kubernetes cluster installed and running.
+- That you have installed the ```kubectl``` command line tool somewhere in your path.
+- That a spark-master service which spins up will be automatically discoverable by your kube DNS impl, as 'spark-master'
```
Comment (zmerlynn) [inline examples/spark/README.md]: This bullet doesn't very clearly state "The SkyDNS add-on is running" to me, whereas 90% of the complaints I've seen, that's what they need to know. I was actually planning on putting up a PR that just checked in the Dockerfile (i.e. tried to resolve `spark-master`, complained that DNS wasn't running.)

Comment (jayunit100) [inline examples/spark/README.md]: Agree, another PR would be great on the docker image itself

**Pull request merged**